python_bindings_jupedsim/bindings_jupedsim.cpp

```diff
@@ -338,6 +342,7 @@ PYBIND11_MODULE(py_jupedsim, m)
             py::arg("model"),
             py::arg("geometry"),
             py::arg("areas"),
+            py::kw_only(),
```

Comment:
I would like all parameters on simulation to require named arguments

Reply:
Addressed both comments in one commit